logstash_async/formatter.py

```diff
@@ -200,7 +200,7 @@ def _serialize(self, message):
         if sys.version_info < (3, 0):
             return json.dumps(message)
         else:
-            return bytes(json.dumps(message, ensure_ascii=self._ensure_ascii), 'utf-8')
+            return json.dumps(message, ensure_ascii=self._ensure_ascii)
```

Comment:
If you like, just remove the version switch and use only your final line for the whole method.
There is no more difference to Python2 then except the `ensure_ascii` option which was simply missing for the Python2 branch.
If not, I will do it after merging.

Reply:
It didn't work for me with `six.u`, so I've used `u"

Reply:
Alright, thanks.